src/stores/walletStore/walletStore.test.tsx

```diff
@@ -167,48 +167,36 @@ describe('walletStore', () => {
   });
 
   test('should connect to a selected provider', async () => {
-    const metamask = mockEIP6963Provider({
-      name: 'MetaMask',
-      uuid: 'metamask-uuid',
-      accounts: ['0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266'],
-    });
```

Comment:
There's a mock metamask connection set up in the `describe` block, so let's just use that instead.